src/features/pages/markets/etf/check-trading/styles.module.scss

```diff
@@ -0,0 +1,8 @@
+@use 'features/styles/theme/theme-mixins' as *;
+@import 'features/styles/theme/variables';
+
+.item_container {
+    @include breakpoints(tablet, max) {
+        box-shadow: rgba(0, 0, 0, 0.1) 0 0 3px 0;
+    }
```

Comment:
Do we need this? and we use mobile first approach.

Reply:
Following the design and copy from the styled components before atomic approach. Will change it to mobile first approach